### PR TITLE
Launch import flow from Password management screen

### PR DESCRIPTION
**Pull request by CDRussell**
Task/Issue URL: https://app.asana.com/0/608920331025315/1208434183978073/f 

### Description
Adds UI which allows users to access the flow for importing passwords directly from Google Password Manager. 

output

### Steps to test this PR

**Empty state**
- [ ] Fresh install
- [ ] Visit `Passwords` screen and verify you see `Import Passwords From Google` button in the empty state (if `WebView` version supports it)

**Pre-import dialog**
- [ ] Tap the button, and verify the bottom sheet looks good

**Success results dialog**
- [ ] Tap `Open Google Passwords` button and complete the flow
- [ ] Verify that you see the success dialog confirming how many passwords were imported. 

**Success results dialog (with duplicates detected)**
- [ ] [If you didn't see any duplicates in the last step] repeat previous step and verify duplicates are mentioned but not imported

**Failure dialog**
It's possible the flow couldn't be completed if the user enabled an old setting in GPM to encrypt passwords with a passphrase. Expecting this to be uncommon, but here's how to test the UI if this happens:

- [ ] Apply [Patch for testing: encrypted with passphrase](https://app.asana.com/0/1208775021175583/1208775021175583/f)
- [ ] Start the import flow, and you'll see the error page
- [ ] Tap the ✖️ button 
- [ ] Verify the results dialog shows an error state

### CODE REVIEW

**Comment by CDRussell** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5098?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5098** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5098?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5097** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5097?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4602** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4602?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4601** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4601?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5096** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5096?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CDRussell and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: autofill/autofill-impl/src/main/res/values/donottranslate.xml [hunk 1]
```diff
@@ -20,4 +20,25 @@
 
     <string name="autofillImportGooglePasswordsWebFlowTitle">Import Google Passwords</string>
     <string name="autofillImportGooglePasswordsSuccessMessage">%1$d passwords imported from Google</string>
+    <string name="autofillImportCsvPasswordsSuccessMessage">%1$d passwords imported from CSV</string>
```
Comment (cmonfortep) [inline autofill/autofill-impl/src/main/res/values/donottranslate.xml:23]: reminder to add instructions here.

File: autofill/autofill-impl/src/main/res/drawable/ic_cross_recolorable_red_24.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M12,22C17.523,22 22,17.523 22,12C22,6.477 17.523,2 12,2C6.477,2 2,6.477 2,12C2,17.523 6.477,22 12,22Z"
+      android:fillColor="#EB102D"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M16.207,9.207C16.598,8.817 16.598,8.183 16.207,7.793C15.817,7.402 15.183,7.402 14.793,7.793L12,10.586L9.207,7.793C8.817,7.402 8.183,7.402 7.793,7.793C7.402,8.183 7.402,8.817 7.793,9.207L10.586,12L7.793,14.793C7.402,15.183 7.402,15.817 7.793,16.207C8.183,16.598 8.817,16.598 9.207,16.207L12,13.414L14.793,16.207C15.183,16.598 15.817,16.598 16.207,16.207C16.598,15.817 16.598,15.183 16.207,14.793L13.414,12L16.207,9.207Z"
+      android:fillColor="#ffffff"/>
```
Comment (cmonfortep) [inline autofill/autofill-impl/src/main/res/drawable/ic_cross_recolorable_red_24.xml:12]: unsure if applies, but best check how this works in both themes, and maybe refer to the actual color.

File: autofill/autofill-impl/src/main/res/drawable/ic_check_recolorable_24.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M12,22C17.523,22 22,17.523 22,12C22,6.477 17.523,2 12,2C6.477,2 2,6.477 2,12C2,17.523 6.477,22 12,22Z"
+      android:fillColor="#21C000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M16.6,8.28C16.998,8.611 17.052,9.202 16.72,9.6L11.72,15.6C11.552,15.802 11.306,15.925 11.042,15.937C10.779,15.948 10.523,15.849 10.337,15.663L7.337,12.663C6.971,12.297 6.971,11.703 7.337,11.337C7.703,10.971 8.297,10.971 8.663,11.337L10.937,13.611L15.28,8.4C15.611,8.002 16.202,7.948 16.6,8.28Z"
+      android:fillColor="#ffffff"
```
Comment (cmonfortep) [inline autofill/autofill-impl/src/main/res/drawable/ic_check_recolorable_24.xml:12]: unsure if applies, but best check how this works in both themes, and maybe refer to the actual color.

File: autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AutofillAuthorizationGracePeriod.kt [hunk 4]
```diff
@@ -40,6 +40,16 @@ interface AutofillAuthorizationGracePeriod {
      */
     fun recordSuccessfulAuthorization()
 
+    /**
+     * Requests an extended grace period. This may extend the grace period to a longer duration.
+     */
+    fun requestExtendedGracePeriod()
```
Comment (cmonfortep) [inline autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AutofillAuthorizationGracePeriod.kt:46]: Can you give me some extra context about what's this extended grace period is for?

File: autofill/autofill-impl/src/main/res/drawable/ic_check_recolorable_24.xml (hunk 3 above)
Comment (CDRussell) [reply autofill/autofill-impl/src/main/res/drawable/ic_check_recolorable_24.xml:12]: Working as intended in both themes

File: autofill/autofill-impl/src/main/res/drawable/ic_cross_recolorable_red_24.xml (hunk 2 above)
Comment (CDRussell) [reply autofill/autofill-impl/src/main/res/drawable/ic_cross_recolorable_red_24.xml:12]: Working as intended in both themes

File: autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AutofillAuthorizationGracePeriod.kt (hunk 4 above)
Comment (CDRussell) [reply autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AutofillAuthorizationGracePeriod.kt:46]: More context here: https://app.asana.com/0/0/1208489120228733/1208627439305609/f

File: autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/AutofillSettingsViewModel.kt [hunk 5]
```diff
@@ -771,6 +793,14 @@ class AutofillSettingsViewModel @Inject constructor(
         }
     }
 
+    fun recordImportGooglePasswordButtonShown() {
+        if (!importGooglePasswordButtonShownPixelSent) {
+            importGooglePasswordButtonShownPixelSent = true
+
+            // pixel to show import button would fire here
```
Comment (CDRussell) [inline autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/AutofillSettingsViewModel.kt:800]: next PR will implement this

**Review by cmonfortep — APPROVED** [review]
LGTM, 👍

**Pull request merged**